### DISCUSSION TITLE
Anyone know how to restore Plover's default settings?

### DISCUSSION THREAD

**Discussion by Register111** [opensteno/plover#1385] [Q&A and Support] [answered]
I accidentally deleted the dictionaries within Plover, and I cannot seem to get them back.  I even deleted the application to try and get it restored. 
TIA

**Comment by user202729** [comment, marked as answer]
Assuming you're using Plover version 4:

* Open "File -> Open config folder"
* Exit Plover
* Delete the plover.cfg file
* Open Plover.

Alternatively, just add them back (there are just 3 files: user.json, commands.json, main.json), or if you haven't restarted Plover, choose "Edit -> Undo".

------

1. It's also possible to navigate to [`CONFIG_DIR`](https://plover.readthedocs.io/en/latest/api/oslayer_config.html#plover.oslayer.config.CONFIG_DIR) -- however, for backward compatibility, the configuration directory might be at `.local/share/plover` on Linux)
2. When plover.cfg is deleted, the configuration directory might change, see above.
